### PR TITLE
Dev sir 1563 refactor deposits

### PR DESCRIPTION
**Pull request by OttoVT**

### CODE REVIEW

File: src/Api.Worker/Api.Worker.csproj [hunk 1]
```diff
@@ -25,9 +25,9 @@
     <PackageReference Include="Swisschain.Sdk.Server" Version="6.8.0" />
     <PackageReference Include="Swisschain.Sirius.Confirmator.MessagingContract" Version="3.0.1-beta" />
     <PackageReference Include="Swisschain.Sirius.Indexer.MessagingContract" Version="4.0.0" />
-    <PackageReference Include="Swisschain.Sirius.Integrations.MessagingContract" Version="8.1.0" />
-    <PackageReference Include="Swisschain.Sirius.VaultAgent.MessagingContract" Version="3.0.1" />
-    <PackageReference Include="Swisschain.Sirius.Brokerage.MessagingContract" Version="4.0.0" />
+    <PackageReference Include="Swisschain.Sirius.Integrations.MessagingContract" Version="9.0.0" />
+    <PackageReference Include="Swisschain.Sirius.VaultAgent.MessagingContract" Version="3.1.0" />
+    <PackageReference Include="Swisschain.Sirius.Brokerage.MessagingContract" Version="5.0.0-beta.2" />
```
Comment (KonstantinRyazantsev) [inline src/Api.Worker/Api.Worker.csproj:30]: Release it

File: src/Api/WebApi/WithdrawalsController.cs [hunk 2]
```diff
@@ -82,7 +82,7 @@ public class WithdrawalsController : ControllerBase
 
             if (string.IsNullOrEmpty(request.Document))
             {
-                ModelState.AddModelError(nameof(request.Document), "Required field.");
+                ModelState.AddModelError("", $"{nameof(request.Document)} required.");
```
Comment (KonstantinRyazantsev) [inline src/Api/WebApi/WithdrawalsController.cs:85]: why?

**Review by KonstantinRyazantsev — COMMENTED** [review]
Let's add deposit type to the filters in the deposits search endpoints (gRPC and REST)

File: src/Api/WebApi/WithdrawalsController.cs (hunk 2 above)
Comment (aushakov) [reply src/Api/WebApi/WithdrawalsController.cs:85]: This controller is very specific. It should be suitable for UI. UI don't know about the document property but it knows properties in the document. The JSON serialization realized in services, view model works with the typed object.

**Comment by KonstantinRyazantsev** [comment]
> Let's add deposit type to the filters in the deposits search endpoints (gRPC and REST)

Will do in the separate task

**Review by KonstantinRyazantsev — APPROVED** [review]

**Pull request merged**